src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -56,15 +86,13 @@ protected override void CustomAttach()
     }
 
     /// <summary>
-    ///   Calculate the momentum of the movement organelle based on
-    ///   angle towards middle of cell
+    ///   Calculate the vector of the thrust force provided by the flagellum based on its orientation.
     /// </summary>
-    private static Vector3 CalculateForce(Hex pos, float momentum)
+    private Vector3 CalculateForce(float momentum)
     {
-        Vector3 organelle = Hex.AxialToCartesian(pos);
-        Vector3 middle = Hex.AxialToCartesian(new Hex(0, 0));
-        var delta = middle - organelle;
-        return delta.Normalized() * momentum;
+        Vector3 forceVector = GetForceVector(organelle.Orientation);
+        forceVector = forceVector * momentum;
```

Comment:
This is unnecessary copy of the data, just put the momentum multiple directly in the return statement